docs/next/guides/formulas/formula-calculation.md

```diff
@@ -3,196 +3,435 @@ title: Formula calculation
+  ['141', '576', '0.63', '12', '=A54*(B54*C54)+D54'],
+  ['169', '429', '0.14', '13', '=A55*(B55*C55)+D55'],
+  ['39', '694', '0.98', '13', '=A56*(B56*C56)+D56'],
+  ['71', '361', '0.3', '13', '=A57*(B57*C57)+D57'],
+  ['148', '540', '0.89', '6', '=A58*(B58*C58)+D58'],
+  ['116', '52', '0.3', '7', '=A59*(B59*C59)+D59'],
+  ['96', '395', '0.28', '7', '=A60*(B60*C60)+D60'],
+  ['35', '222', '0.86', '13', '=A61*(B61*C61)+D61'],
+  ['16', '430', '0.8', '8', '=A62*(B62*C62)+D62'],
+  ['194', '357', '0.72', '9', '=A63*(B63*C63)+D63'],
+  ['24', '498', '0.7', '7', '=A64*(B64*C64)+D64'],
+  ['170', '142', '0.52', '5', '=A65*(B65*C65)+D65'],
+  ['184', '614', '0.68', '9', '=A66*(B66*C66)+D66'],
+  ['153', '524', '0.15', '9', '=A67*(B67*C67)+D67'],
+  ['88', '620', '0.39', '15', '=A68*(B68*C68)+D68'],
+  ['57', '452', '0.11', '6', '=A69*(B69*C69)+D69'],
+  ['62', '493', '0.03', '11', '=A70*(B70*C70)+D70'],
+  ['123', '431', '0.75', '15', '=A71*(B71*C71)+D71'],
+  ['77', '113', '0.63', '12', '=A72*(B72*C72)+D72'],
+  ['199', '208', '0.07', '6', '=A73*(B73*C73)+D73'],
+  ['149', '514', '0.42', '12', '=A74*(B74*C74)+D74'],
+  ['191', '334', '0.78', '13', '=A75*(B75*C75)+D75'],
+  ['150', '643', '0.32', '11', '=A76*(B76*C76)+D76'],
+  ['130', '721', '0.62', '5', '=A77*(B77*C77)+D77'],
+  ['179', '517', '0.1', '8', '=A78*(B78*C78)+D78'],
+  ['98', '31', '0.01', '10', '=A79*(B79*C79)+D79'],
+  ['175', '509', '0.7', '11', '=A80*(B80*C80)+D80'],
+  ['11', '569', '0.37', '7', '=A81*(B81*C81)+D81'],
+  ['184', '630', '0.19', '6', '=A82*(B82*C82)+D82'],
+  ['27', '165', '0.51', '13', '=A83*(B83*C83)+D83'],
+  ['186', '417', '0.85', '15', '=A84*(B84*C84)+D84'],
+  ['20', '741', '0.76', '11', '=A85*(B85*C85)+D85'],
+  ['153', '640', '0.28', '6', '=A86*(B86*C86)+D86'],
+  ['161', '542', '0.43', '14', '=A87*(B87*C87)+D87'],
+  ['98', '344', '0.77', '6', '=A88*(B88*C88)+D88'],
+  ['30', '400', '0.71', '13', '=A89*(B89*C89)+D89'],
+  ['73', '91', '0.78', '8', '=A90*(B90*C90)+D90'],
+  ['158', '72', '0.66', '12', '=A91*(B91*C91)+D91'],
+  ['122', '35', '0.32', '15', '=A92*(B92*C92)+D92'],
+  ['33', '99', '0.92', '9', '=A93*(B93*C93)+D93'],
+  ['107', '538', '0.75', '10', '=A94*(B94*C94)+D94'],
+  ['15', '563', '0.32', '6', '=A95*(B95*C95)+D95'],
+  ['168', '572', '0.71', '6', '=A96*(B96*C96)+D96'],
+  ['135', '217', '0.49', '12', '=A97*(B97*C97)+D97'],
+  ['11', '595', '0.03', '13', '=A98*(B98*C98)+D98'],
+  ['41', '739', '0.88', '11', '=A99*(B99*C99)+D99'],
+  ['144', '289', '0.87', '13', '=A100*(B100*C100)+D100'],
+  ['Sum', 'Average', 'Average', 'Sum', 'Sum'],
+  ['=SUM(A1:A100)', '=AVERAGE(B1:B100)', '=AVERAGE(C1:C100)', '=SUM(D1:D100)', '=SUM(E1:E100)']
 ];
 
-const hot = new Handsontable(container, {
-  data,
-  colHeaders: true,
-  rowHeaders: true,
-  contextMenu: true,
-  licenseKey: 'non-commercial-and-evaluation',
+const container = document.getElementById('example-data-grid');
+new Handsontable(container, {
+  data: data,
```

Comment:
There is lack of `licenseKey: 'non-commercial-and-evaluation',`. Could you please add it?

Reply:
d4905055c0f6487bfc107b5ff3b7a6bb796193ab